### ISSUE TITLE
`failed to resolve: could not find `async_sync` in `cached``

### ISSUE THREAD

**Issue by gitmalong** [jaemk/cached#172]
```toml
cached = { version = "0.45.1", features = ["wasm"] }
```

```rust
#[cached(
    type = "SizedCache<String, Option<Decimal>>",
    create = "{ SizedCache::with_size(100) }",
    convert = r#"{ format!("{:?}{:?}{:?}{:?}", asset, fiat, day, price_type) }"#
)]
async fn ccached_price(
    p: &impl PriceDisoveryService,
    asset: Asset,
    fiat: String,
    day: DateTime<Utc>,
    price_type: PriceType,
) -> Option<Decimal> {
    p.price(asset, fiat, day, price_type).await
}
```

error

```rust
36 | / #[cached(
37 | |     type = "SizedCache<String, usize>",
38 | |     create = "{ SizedCache::with_size(100) }",
39 | |     convert = r#"{ format!("{:?}{:?}{:?}{:?}", asset, fiat, day, price_type) }"#
40 | | )]
   | |__^ could not find `async_sync` in `cached`
```

using this in a WASM CSR app based on https://github.com/leptos-rs/leptos/tree/main/examples/tailwind_csr_trunk

**Comment by gitmalong** [comment]
This fixed my issue: `cached = { version = "0.45.1", features = ["wasm", "async"] }`

**Comment by gitmalong** [comment]
This fixed my issue `cached = { version = "0.45.1", features = ["wasm", "async", "proc_macro"] }`

**Comment by FilipAndersson245** [comment]
Async was removed as default feature in latest release, think there is a PR to update the docs.
